pkg/minikube/machine/drivers/none/none.go

```diff
@@ -153,6 +153,7 @@ func (d *Driver) SetConfigFromFlags(flags drivers.DriverOptions) error {
 }
 
 func (d *Driver) Start() error {
+	os.Mkdir("/usr/lib/systemd/system/",os.FileMode(0755))
```

Comment:
Would you mind switching this to use MkdirAll and add an error check?

This should work:
```go
	if err := os.MkdirAll("/usr/lib/systemd/system/", os.FileMode(0755)); err != nil {
		return err
	}
```